ras_party/models/models.py

```diff
@@ -83,21 +83,35 @@ def _get_respondents_associations(respondents):
             associations.append(respondent_dict)
         return associations
 
-    def to_business_dict(self):
-        d = {
-            'id': self.party_uuid,
-            'sampleUnitRef': self.business_ref,
-            'sampleUnitType': self.UNIT_TYPE,
-            'associations': self._get_respondents_associations(self.respondents),
-        }
-        return dict(d, **self.attributes)
+    def to_business_dict(self, summary=False):
```

Comment:
Is it worth splitting this logic out into `to_business_dict` and `to_business_summary_dict` to create more cohesive easier to test methods?